aea/configurations/base.py

```diff
@@ -1014,22 +1069,24 @@ def package_dependencies(self) -> Set[ComponentId]:
     @property
     def json(self) -> Dict:
         """Return the JSON representation."""
-        return {
-            "name": self.name,
-            "author": self.author,
-            "version": self.version,
-            "license": self.license,
-            "aea_version": self.aea_version,
-            "fingerprint": self.fingerprint,
-            "fingerprint_ignore_patterns": self.fingerprint_ignore_patterns,
-            "protocols": sorted(map(str, self.protocols)),
-            "contracts": sorted(map(str, self.contracts)),
-            "dependencies": self.dependencies,
-            "handlers": {key: h.json for key, h in self.handlers.read_all()},
-            "behaviours": {key: b.json for key, b in self.behaviours.read_all()},
-            "models": {key: m.json for key, m in self.models.read_all()},
-            "description": self.description,
-        }
+        return OrderedDict(
+            {
+                "name": self.name,
+                "author": self.author,
+                "version": self.version,
+                "description": self.description,
+                "license": self.license,
+                "aea_version": self.aea_version,
+                "fingerprint": self.fingerprint,
+                "fingerprint_ignore_patterns": self.fingerprint_ignore_patterns,
+                "protocols": sorted(map(str, self.protocols)),
```

Comment:
In spirit of above, swap contracts and protocols order, then "behaviours", "handlers", "models", "dependencies"